doc/rules/prefer-single-widget-per-file.md

```diff
@@ -0,0 +1,79 @@
+# Prefer single widget per file
+
+## Rule id
+
+prefer-single-widget-per-file
+
+## Description
+
+Warns when a file contains more than a single widget.
+
+Ensures that files have a single responsibility so that that widget each exist in their own file.
```

Comment:
`so that that widget each exist in their own file.` -> `so that each widget exists in its own file.`

Reply:
fixed